projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,65 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+Using Store with Angular Elements is similar to using it with standard Angular Component.
+
+## Tutorial
+
+The following tutorial shows you how to manage the state of a counter, and how to select and display it within an Angular Elmements and use the same within an Angular Project and an static page. Try the <live-example name="store-elements" downloadOnly>Anguler Elements Project</live-example>.
+
+1. Create a new application by running the below command
+```sh
+ng new ngRxElementDemo --createApplication=false
+```
+2. Create a new Angular Library using below command
+```sh
+ng g library my-counter
+```
+3. Add code mentioned in [Getting Started Guide](guide/store#installation) to new created.
+4. Rename `app.module.ts` to `counter.module.ts` file and add the below code.
+<code-example header="src/lib/counter.module.ts" path="store-elements/projects/my-counter/src/lib/app/counter.module.ts">
+</code-example>
+5. Add 2 application within the project using the below command
+```sh
+ng g application elementApp
+ng g application counterelement
+```
+6. Within elementApp add the below code in `app.module.ts`.
```

Comment:
Same as previously mentioned, provide some more context as to why the user is to add the code.